README.md

```diff
@@ -1,37 +1,37 @@
+Although cryptocurrency is the future, there are still many challenges to overcome. One of them is providing an easy way to manage multiple different types of coins. Although we already have some exchange services, they can be a bit difficult to use and require many steps to perform the actual transfer, and they require you to trust a centralized third party and/or a centralized ERC20 token smart contract. With this suite of smart contracts, we aim to provide a decentralized way of transferring tokens between MultiversX and Ethereum.  
 
-For this to be truly decentralized, we're using a multisig smart contract. More info on that [here](https://github.com/ElrondNetwork/elrond-wasm-rs/blob/master/contracts/examples/multisig/README.md).  Basically, we're going to have a set of relayers that validate the transaction. You could think of it as a mini-blockchain, whose sole purpose is to handle cross chain transactions.  
+For this to be truly decentralized, we're using a multisig smart contract. More info on that [here](https://github.com/multiversx/mx-sdk-rs/blob/master/contracts/examples/multisig/README.md).  Basically, we're going to have a set of relayers that validate the transaction. You could think of it as a mini-blockchain, whose sole purpose is to handle cross chain transactions.  
 
-To be able to transfer the tokens, we make use of a concept known as "Wrapped Tokens". Basically, you don't transfer native EGLD or ETH tokens, but instead you lock the native tokens in a contract and generate "Wrapped" versions of them. On Elrond, we are going to use ESDT for this purpose. On Ethereum, we're most likely going to use something like ERC20.  
+To be able to transfer the tokens, we make use of a concept known as "Wrapped Tokens". Basically, you don't transfer native EGLD or ETH tokens, but instead you lock the native tokens in a contract and generate "Wrapped" versions of them. On MultiversX, we are going to use ESDT for this purpose. On Ethereum, we're most likely going to use something like ERC20.  
 
-Now you might ask, how is this any different? There already is an ERC20 style contract on Ethereum for EGLD! The main difference is you won't have to go through exchange services at all, and at some point, we'll likely have this integrated fully into our Maiar application. And there's one more important difference: This is decentralized. There isn't a single contract owner doing all the work.  We have a set of trusted accounts that will handle the transactions.  
+Now you might ask, how is this any different? There already is an ERC20 style contract on Ethereum for EGLD! The main difference is you won't have to go through exchange services at all, and at some point, we'll likely have this integrated fully into our Mobile Wallet application. And there's one more important difference: This is decentralized. There isn't a single contract owner doing all the work.  We have a set of trusted accounts that will handle the transactions.  
```

Comment:
xPortal instead Mobile Wallet ?

Reply:
Dan Popa said to keep it as Mobile Wallet untill the laucnh of xPortal